### ISSUE TITLE
Any plan for JavaFX version for mobile, iphone etc?

### ISSUE THREAD

**Issue by shashaanktulsyan** [bardsoftware/ganttproject#1343]
Hi,

Any plans for porting this to JavaFX and then porting it to other platforms so that it may be used on mobile etc. May be that would be too much to ask for, but it would be definitely good to have.
Any such plans?

**Comment by dbarashev** [comment]
It is impossible to "port" GanttProject to JavaFX. It is possible to rewrite it from scratch. That's a few months of full-time work to produce application which for 90% of users does exactly the same what the existing app does, but is expected to have more bugs because it is, you know, new and not tested. No, at the moment we're not feeling brave enough for such adventure.

For Android-based devices you may want to try [Ganttman](http://ganttman.com) which can read GanttProject files.

**Comment by shashaanktulsyan** [comment]
Yes I can very well understand. Even I couldn't gather courage and be brave
enough to volunteer to contribute. I am guess how work intensive it would
be. Thanks for the android project link. Any plans of having a cloud based
solution so that the android app and desktop app can be synced ? That would
be nice. And it should be fairly easy. This is something I could volunteer
for.
